optimum/onnxruntime/modeling_ort.py

```diff
@@ -884,6 +1323,60 @@ def prepare_inputs_for_generation(self, input_ids: torch.LongTensor, **kwargs) -
             inputs["attention_mask"] = kwargs["attention_mask"]
```

Comment:
I couldn't comment above, but do we need to adjust the `prepare_inputs_for_generation`, to work with `.generate`?

Reply:
I don't think so, as the preparation will be done while calling forward during the generation.